source/release-notes/5.0.txt

```diff
@@ -48,6 +48,18 @@ to specify a list of variables that can be used elsewhere in the
 aggregation pipeline. This allows you to improve command readability by
 separating the variables from the query text.
 
+$sampleRate
+```````````
+
+MongoDB 5.0 adds the :pipeline:`$sampleRate` method to
+probabilistically select documents from a pipeline at a given rate. 
+
+$rand
+`````
+New to MongoDB 5.0 (and 4.4.2), the :expression:`$rand` method will
+generate a random float value between 0 and 1 each time it is called.
+The new :pipeline:`$sampleRate` method is based on ``$rand``.
```

Comment:
:pipeline:`$sampleRate` should be :expression:`$sampleRate`, no? Please also change this in the $sampleRate header just above. I think this is maybe more of DOCS-14053?

Reply:
Done